lms/envs/devstack.py

```diff
@@ -15,6 +15,7 @@
 SITE_NAME = 'localhost:8000'
 # By default don't use a worker, execute tasks as if they were local functions
 CELERY_ALWAYS_EAGER = True
+HTTPS = 'off'
```

Comment:
Looking at the other settings files, we probably want to make the corresponding changes to `cms`, as well.

Can you add this override to `cms/envs/dev.py` and `cms/envs/devstack.py`?